<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="ATHULYA-HARIKRISHNAN" failures="1" tests="1" name="Test5" time="71.45" errors="0" timestamp="2025-03-13T12:49:52 IST">
  <testcase classname="testPackage.VerificationClass" name="verify" time="0.019">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testPackage.VerificationClass.verify(VerificationClass.java:28)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
at java.base/java.lang.Thread.run(Thread.java:1583)
... Removed 17 stack frames]]>
    </failure>
  </testcase> <!-- verify -->
</testsuite> <!-- Test5 -->
